File: basic/inline variable/python/src/example_2.py
"""
Contexto: Estás construyendo un programa que procesa una lista de números y realiza ciertos cálculos con ellos.
Tienes una variable intermedia llamada promedio que almacena el promedio de la lista de números.
"""
def calcular_promedio_(numeros: list[int | float]) -> int | float:
    suma = sum(numeros)
    cantidad = len(numeros)
    promedio = suma / cantidad  # Refactoriza esta variable intermedia
    return promedio

calcular_promedio = lambda numeros: sum(numeros) / len(numeros)

File: basic/inline variable/javascript/example2.js
/*
Contexto: Estás creando una aplicación de seguimiento de gastos personales.
Tienes una variable intermedia llamada totalGastos que almacena el resultado del cálculo de los gastos mensuales.
*/

const calcularGastosMensuales_ = (
    gastosAlimentacion, gastosTransporte, gastosEntretenimiento) => {
        const totalGastos = gastosAlimentacion + gastosTransporte + gastosEntretenimiento;  // Refactoriza esta variable intermedia
        return totalGastos;
};

const calcularGastosMensuales = (
        gastosAlimentacion, gastosTransporte, gastosEntretenimiento
    ) => gastosAlimentacion + gastosTransporte + gastosEntretenimiento;

module.exports = {calcularGastosMensuales_, calcularGastosMensuales};
